extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -1189,6 +1188,16 @@ public static Token fromAudience(String... audience) {
         @ConfigItem(defaultValue = "true")
         public boolean allowOpaqueTokenIntrospection = true;
 
+        /**
+         * Indirectly verify that the opaque (binary) access token is valid by using it to request UserInfo.
+         * Opaque access token is considered valid if the provider accepted this token and returned a valid UserInfo.
+         * You should only enable this option if the opaque access tokens have to be accepted but OpenId Connect
+         * provider does not have a token introspection endpoint.
+         * This property will have no effect when JWT tokens have to be verified.
+         */
+        @ConfigItem(defaultValue = "false")
+        public boolean verifyOpaqueAccessTokenWithUserInfo;
```

Comment:
Here is one last suggestion to rename it here and everywhere it is referenced. Please try `verifyAccessTokenWithUserInfo`  - the opaque token restriction will remain in the docs and the code. But I've been thinking, what if for ex GitHub decides to use JWT tokens but will still no JWK or token introspection endpoint are available and then the `opaque` in the name will become a problem